packages/core/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * as Utils from "./utils";
+export * as Prismic from "./prismic";
+export * as Models from "./models";
+export * as Libraries from "./libraries";
+export * as Mocks from "./libraries/mocks";
+export * as CONSTS from "./consts";
+export * as NodeUtils from "./node-utils";
```

Comment:
should this export stuff from NodeUtils ?
I mean if the front-end import stuff from '@slicemachine/core' then this might cause an issue again.
(But this might be related to the next ticket :)

Reply:
Yeah I was thinking about that, but the dependancies would mean that `libraries` and `prismic` would also have to be removed too :/